dask/array/core.py

```diff
@@ -1307,6 +1307,10 @@ def ravel(self):
 
     flatten = ravel
 
+    @wraps(np.choose)
+    def choose(self, choices):
+        return elemwise(variadic_choose, self, *choices)
```

Comment:
Just for completeness, the argument for the function calling the method would be to handle inheritance from Dask Array. That said, I have no strong preferences here either and it can always be changed if there is a need.